services/data_plane_manager/src/main/java/com/futurewei/alcor/dataplane/service/impl/DpmServiceImpl.java

```diff
@@ -41,10 +46,20 @@
     private static final boolean USE_PULSAR_CLIENT = false;
 
     private int goalStateMessageVersion;
+    private DataPlaneManagerRestClient dataPlaneManagerRestClient;
+
+    @Value("${zetaGateway.enabled}")
```

Comment:
Here we can set a default value as FALSE, if the specific config doesn't show up in application.properties.

Reply:
default value FALSE added to @Value("${zetaGateway.enabled:false}")